server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/measure/MeasureDtoToMeasure.java

```diff
@@ -123,21 +123,11 @@
   }
 
   private static boolean hasAnyVariation(MeasureDto measureDto) {
-    for (int i = 1; i < 6; i++) {
-      if (measureDto.getVariation(i) != null) {
-        return true;
-      }
-    }
-    return false;
+    return measureDto.getVariation() != null;
   }
 
   private static MeasureVariations createVariations(MeasureDto measureDto) {
-    return new MeasureVariations(
-      measureDto.getVariation(1),
-      measureDto.getVariation(2),
-      measureDto.getVariation(3),
-      measureDto.getVariation(4),
-      measureDto.getVariation(5));
+    return new MeasureVariations(measureDto.getVariation());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "MeasureVariations"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)